docs/data/joy/customization/using-css-variables/using-css-variables.md

```diff
@@ -1,81 +1,97 @@
 # Using CSS variables
 
-<p class="description">Learn how to use CSS variables to customize Joy UI's components.</p>
+<p class="description">Learn how to use CSS variables to customize Joy UI components.</p>
 
-## Theme object
+## Introduction
```

Comment:
Another thought: is this header necessary? If we dropped "Introduction" and "Usage" from the header hierarchy here, we could bump up all of those h4s into h3s, and then they would get added to the TOC nav. It feels like everything on the page is technically about "usage."

Reply:
That makes sense! 🤙

Reply:
That makes sense! 🤙